src/FsAutoComplete.BackgroundServices/Program.fs

```diff
@@ -125,11 +125,27 @@ type BackgroundServiceServer(state: State, client: FsacClient) =
     let mutable latestRuntimeVersion = lazy None
     //TODO: does the backgroundservice ever get config updates?
     do
+      let allowedVersionRange =
+        let maxVersion = System.Environment.Version.Major + 1
```

Comment:
this version range will be 5.somethhing on .net 5 and 6.something on .net 6, so we should ensure that the SDK/Runtime we load for FSX is within that bound.

This still floats to the highest in that bracket, but we can very easily allow some user customization here and just change the way we create the Range.